Initial Layout Calendar

den-stronin opened the pull request:
- [DEMO LINK](https://den-stronin.github.io/layout_calendar/)
Not found - [TEST REPORT LINK](https://den-stronin.github.io/layout_calendar/report/html_report/)

den-stronin committed:
Initial Layout Calendar

den-stronin committed:
Add Readme.md

DarkMistyRoom commented on src/styles/blocks/calendar.scss:
```diff
@@ -0,0 +1,24 @@
+.calendar {
+  margin: $day-translateY auto;
```
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

DarkMistyRoom commented on src/styles/blocks/day.scss:
```diff
@@ -0,0 +1,24 @@
+  border: 1px solid black;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  transition: transform 0.5s, background-color 0.5s;
```
move 0.5s to a variable

DarkMistyRoom commented on src/styles/blocks/day.scss:
```diff
@@ -0,0 +1,24 @@
+  background-color: #eee;
+
+  &:hover {
+    cursor: pointer;
```
move cursor: pointer to `day `selector

DarkMistyRoom commented on src/styles/blocks/day.scss:
```diff
@@ -0,0 +1,24 @@
+.day {
```
better to make the `day `a `calendar `element, because it is unlikely to be used without the `calendar `in the context of our project

DarkMistyRoom reviewed (changes requested):
Good job, just few comments

den-stronin committed:
Modified blocks & utils

den-stronin commented:
1. "Don't add external styles ..." - Completed. Added the new 'main' container.
2. "move 0.5s to a variable" - Completed.
3. "move cursor: pointer to day selector" - Completed.
4. "better to make the day a calendar element..." - Completed. Modified html & SCSS files.

ivbutsykin commented on src/styles/blocks/calendar.scss:
```diff
@@ -0,0 +1,45 @@
+    font-size: 30px;
+    cursor: pointer;
+    transition: transform $day-transition, background-color $day-transition;
+    background-color: #eee;
```
Will be nice to move all colors in separate variables

den-stronin committed:
Added separate color variables

den-stronin commented:
"Will be nice ... separate variables" - Completed. 
Will I start new "Re-request review"?
